src/Hakyll/ShortcutLinks.hs

```diff
@@ -10,6 +10,27 @@ package in websites generated by [hakyll](https://hackage.haskell.org/package/ha
 
 The flexible interface allows to use the supported huge collection of shortcuts
 along with using custom ones.
+
+Here is a few examples of the `@github` shortcut:
+
+- Link to a user:
++====================================+==============================================+
+|              Shortcut              |                Plain markdown                |
++====================================+==============================================+
+| [foo](@github)                     | [foo](https://github.com/foo)                |
++------------------------------------+----------------------------------------------+
+| [foo Github profile](@github(foo)) | [foo Github profile](https://github.com/foo) |
++------------------------------------+----------------------------------------------+
+
+- Link to a repository:
++===================================+=============================================+
+|             Shortcut              |               Plain markdown                |
++===================================+=============================================+
+| [bar](@github:foo)                | [bar](https://github.com/foo/bar)           |
++-----------------------------------+---------------------------------------------+
+| [Github Source](@github(foo):bar) | [Github Source](https://github.com/foo/bar) |
++-----------------------------------+---------------------------------------------+
+
```

Comment:
Sadly, this is not working there is a number of syntax errors, yeah, Haddock is not that straightforward :disappointed: And also there is no escaping, so it won't be displayed the way we need it.

I understand, that you might not be interested in looking through the Haddock docs, and actually checking how the documentation you write looks like (by running `cabal haddock`), so I made working piece, you can just paste it :slightly_smiling_face: :

```
Here is a few examples of the `@github` shortcut:

- Link to a user:

+----------------------------------------+----------------------------------------------------+
|              Shortcut                  |                Plain markdown                      |
+========================================+====================================================+
| @[foo]\(\@github)@                     | @[foo]\(https:\/\/github.com\/foo)@                |
+----------------------------------------+----------------------------------------------------+
| @[foo Github profile]\(\@github(foo))@ | @[foo Github profile]\(https:\/\/github.com\/foo)@ |
+----------------------------------------+----------------------------------------------------+

- Link to a repository:

+---------------------------------------+----------------------------------------------------+
|             Shortcut                  |               Plain markdown                       |
+=======================================+====================================================+
| @[bar]\(\@github:foo)@                | @[bar]\(https:\/\/github.com\/foo\/bar)@           |
+---------------------------------------+----------------------------------------------------+
| @[Github Source]\(\@github(foo):bar)@ | @[Github Source]\(https:\/\/github.com\/foo\/bar)@ |
+---------------------------------------+----------------------------------------------------+

```

If you have any questions you can ask me, I'll be happy to help :slightly_smiling_face:

Reply:
No worries , i'm happy to help and learn, it's my first time writing Haddock docs. 
i'll update the Haddock Table. 

Thanks for your help and support  😄